modules/local/atacreads/dumpreads.nf

```diff
@@ -0,0 +1,38 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from '../functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process DUMPREADS {
+    tag "${meta.id}"
+    label 'process_medium'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "anaconda::gawk=5.1.0" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/gawk:5.1.0"
+    } else {
+        container "quay.io/biocontainers/gawk:5.1.0"
+    }
+
+    input:
+    tuple val(meta), path(bed)
+
+    output:
+    tuple val(meta), path("${meta.id}/*.shrt.vip.bed") , emit: peak
+    path  "*.version.txt"                                                , emit: version
+
+    script:
+    def software = "awk"
+    def prefix   = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
+    def outdir   = "${meta.id}"
+    """
+    mkdir -p $outdir
+    cat $bed | zcat | awk -v setname=${prefix} -v outdir=${outdir}  -F \$"\t"  'BEGIN {OFS=FS};{print \$1,\$2,\$3 > outdir"/"setname"."\$1".shrt.vip.bed"}'
+
+    echo \$(awk --version 2>&1) | sed -e "s/GNU Awk //g; s/, API.*\$//" > ${software}.version.txt
```

Comment:
```suggestion
    def software = "awk"
    def prefix   = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
    """
    zcat $bed | \\
        awk -F "\t" 'BEGIN { OFS=FS } { print \$1,\$2,\$3 > ${prefix}"."\$1".shrt.vip.bed" }'

    echo \$(awk --version 2>&1) | sed -e "s/GNU Awk //g; s/, API.*\$//" > ${software}.version.txt
```
To make this a bit more readable for me, I trimmed down the command a bit to reduce the number of variables.
The output path above would also need to be updated too. 
What was the $ in `-F \$"\t"` doing?

Reply:
`-F \$"\t"` should be a typo. I will correct this.